__tests__/pubsub/subscriptions/test-topic.example.subscription.ts

```diff
@@ -0,0 +1,14 @@
+import { SubscriberObject } from '@honestfoodcompany/pubsub';
+
+const TestSubscription: SubscriberObject = {
+  topicName: 'test-topic',
+  subscriptionName: 'test-topic.example.subscription',
+  description: 'Just a test subscription',
+  async handleMessage(message: any): Promise<void> {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8106467)
```suggestion
  async handleMessage(message: unknown): Promise<void> {
```